staging/cse/windows/provisioningscripts/kubeletstart.ps1

```diff
@@ -66,6 +76,22 @@ if ($global:ContainerRuntime -eq "containerd") {
     $KubeletArgList += @("--container-runtime=remote", "--container-runtime-endpoint=npipe://./pipe/containerd-containerd")
 }
 
+# Update args to get both IPs from NIC using the default route 0.0.0.0/0 if IsDualStackEnabled
+# and the --node-ip argument is not present in the existing kubelet arg list.
+if ($global:IsDualStackEnabled -and !(Test-NodeIPArgExists $KubeletArgList)) {
+    $defaultRouteIfIndex = (Get-NetRoute -DestinationPrefix 0.0.0.0/0).ifIndex
+
+    # use addresses from dhcp and sort by family so IPv4 is always first
```

Comment:
Somewhat nervous that we do this in windows but not linux. like to be consistent. another possiblity is using an annottion for dnc-rc and leaving this argument alone. Fine with this as short term mitigation but we might want to say this is only realyl needed by dnc-rc.

Reply:
yeah I think adding an annotation would work, and +1 to putting it in Linux. imo I like using this argument for kubelet because then the node is created with both IPs and there's no period of waiting for the IPv6 address. currently, even in linux, the node gets created with just an IPv4 address and then after an arbitrary amount of time the node is patched with the IPv6 address. the inconsistency concern would be if we use --node-ip for dual-stack only but single-stack still just relies on kubelet defaulting